Sources/Growth/P3AUmaExtensions.swift

```diff
@@ -52,3 +53,33 @@ public func UmaHistogramRecordValueToBucket(
   }
   UmaHistogramExactLinear(name, answer, buckets.count + 1)
 }
+
+/// Adds a sample to record info around the last time a feature was used.
+///
+/// By default this uses the 7 standard buckets used in many P3A questions but alternative ones can be passed
+/// in to use if needed.
+public func UmaHistogramRecordLastFeatureUsage(
+  _ name: String,
+  option: Preferences.Option<Date?>,
+  alternativeBuckets: [Bucket]? = nil
+) {
+  let calendar = Calendar.current
```

Comment:
No need to hardcode gregorian cal?

Reply:
Probably not but for consistency sake I will switch it to use gregorian for everything